packages/edit-site/src/components/page-pages/index.js

```diff
@@ -37,6 +37,10 @@ const defaultConfigPerViewType = {
 	},
 };
 
+// DEFAULT_STATUSES is intentionally sorted. Items do not have spaces in between them.
+// The reason for that is to match the default statuses coming from the endpoint (entity request).
+export const DEFAULT_STATUSES = 'draft,future,pending,private,publish'; // All statuses but 'trash'.
```

Comment:
By changing this string (e.g.: remove one of the items), two requests will be dispatched to the pages endpoint. The same will happen if the site provides custom statuses.